Gems/AWSCore/Code/Tools/ResourceMappingTool/model/constants.py

```diff
@@ -24,6 +24,7 @@
 # Default client&server config file name
 RESOURCE_MAPPING_CONFIG_FILE_NAME_SUFFIX: str = "_aws_resource_mappings.json"
 RESOURCE_MAPPING_DEFAULT_CONFIG_FILE_NAME: str = "default" + RESOURCE_MAPPING_CONFIG_FILE_NAME_SUFFIX
+RESOURCE_MAPPING_DEFAULT_CONFIG_FILE_REGION: str = "us-west-2"
```

Comment:
minor: this should be us-east-1 aka US standard region.

Reply:
Will make the change.